integration-tests/Cargo.toml

```diff
@@ -102,6 +106,7 @@ std = [
 	"sp-consensus-grandpa/std",
 	"sp-core/std",
 	"sp-runtime/std",
+	"tracing-subscriber/std",
 	"westend-runtime-constants?/std",
 	"westend-runtime?/std",
 	"xcm-executor/std",
```

Comment:
These dependencies were added when trying to find a compilation error. Leaving them in as they might cause problems in the future.